### ISSUE TITLE
Lock version of Web5 for Phase 2

### ISSUE THREAD

**Issue by ALRubinger** [TBD54566975/developer.tbd.website#397]
From Discord:
https://discord.com/channels/937858703112155166/969272658501976117/1096373020093861949

What's the version of Web5 we're using for Phase 2? And what's the direct link we'll advise to pull it from the CDN?

Please assign to me when answered? I'll update the Quickstart, for instance:

```
import { Web5 } from 'https://cdn.jsdelivr.net/npm/@tbd54566975/web5@0.3.1/dist/browser.min.js'
```

I'll also update Devin's example: https://github.com/TBD54566975/web5-quickstart-widgets/blob/main/index.html#L172

**Comment by ALRubinger** [comment]
As decided w/ @csuwildcat:

* We'll continue with phase-2 publish today, merging to main when all blockers are cleared
* No splashy announcement about it on Discord or elsewhere quite yet
* OSE will get us a version number to lock docs against - that will resolve @ALR | TBD blocker https://github.com/TBD54566975/developer.tbd.website/issues/397
* OSE is working on a set of API changes to land early next week. 
* So we'll fast follow with a docs update to upgrade the version of web5-js that we're documenting, and also update the docs to match.
* That process will actually be really helpful - it'll show everyone on the team all the things that need to get updated when we push new versions.

**Comment by ALRubinger** [comment]
Surfacing what likely needs to change when we update the version:

https://github.com/TBD54566975/developer.tbd.website/issues/392#issuecomment-1509238801

There may be API changes in the new version, and we'll have to update `web5-quickstart-widgets`, make sure that's working, then bring that code into the Quickstart.

No updating Quickstart code from anything that's not verified as working in the `web5-quickstart-widgets repo.

**Comment by ALRubinger** [comment]
@mistermoe cut `0.4.0`!

For NPM: `npm install @tbd54566975/web5@0.4.0`
For CDN: `import { Web5 } from 'https://cdn.jsdelivr.net/npm/@tbd54566975/web5@0.4.0/dist/browser.mjs';`

This resolves the OSE dependency and now what's left in this issue is updating examples and docs to match the `0.4.0` upgrade.

**Comment by ALRubinger** [comment]
Confirmed this is working in `web5-quickstart-widgets` as a drop-in upgrade w/ no code changes necessary: 

https://github.com/TBD54566975/web5-quickstart-widgets/issues/1

**Comment by ALRubinger** [comment]
Now updating the Quickstart docs and installation instructions to put the `0.4.0` version in place.

**Comment by ALRubinger** [comment]
Closed by #405
